test_howdoi.py

```diff
@@ -88,7 +93,7 @@ def test_answer_links_using_all_option(self):
             self.assertNotEqual(re.match('.*http.?://.*questions/\d.*', response, re.DOTALL), None)
 
     def test_position(self):
-        query = self.queries[0]
+        query = random.choice(self.queries)
         first_answer = self.call_howdoi(query)
         not_first_answer = self.call_howdoi(query + ' -p5')
         self.assertNotEqual(first_answer, not_first_answer)
```

Comment:
What's the difference of getting a random query and getting the first one? This could mess up testing because one query could fail and other could pass (i.e. non-deterministic outputs). Also, not sure if it's related.

Reply:
yeah please just use the first. I'm not sure if every query is going to have 5 answers but I know the first one does.